packages/@aws-cdk/aws-codebuild/test/integ.github-webhook-batch.ts

```diff
@@ -0,0 +1,29 @@
+import * as cdk from '@aws-cdk/core';
+import * as codebuild from '../lib';
+
+class TestStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string) {
+    super(scope, id);
+
+    const source = codebuild.Source.gitHub({
+      owner: 'aws',
+      repo: 'aws-cdk',
+      reportBuildStatus: false,
```

Comment:
I _think_ the default for this is `false`, so no reason to set it.

Reply:
```ts
  /**
   * Whether to send notifications on your build's start and end.
   *
   * @default true
   */
  readonly reportBuildStatus?: boolean;
  ```
  
It's true by default, and also `false` in the `integ.github` test, so I'll leave it for now. Let me know if you still want it switching